sequence/src/main/java/com/vitreoussoftware/bioinformatics/sequence/encoding/IupacEncodingScheme.java

```diff
@@ -5,29 +5,48 @@
 
 
 /**
```

Comment:
This class shares a lot of code with the ExpandedIupacEncodingScheme.  Opportunity for sharing or not worth? Also, same comments from that class apply here.